Dockerfile

```diff
@@ -23,12 +23,13 @@ RUN apt-get update \
 	libcsmith-dev \
 	creduce \
 	gcc-multilib \
-	clang-6.0 \
-	llvm-6.0 \
+	clang \
+	llvm \
+	lld \
+	wabt \
```

Comment:
Does this allow the Rust `wabt` crate to build faster? If not, I would prefer not to add things that aren't used in CI.